src/main/java/com/minecolonies/coremod/colony/managers/StatisticAchievementManager.java

```diff
@@ -34,14 +34,28 @@
     private int felledTrees       = 0;
     private int plantedSaplings   = 0;
 
+    /**
+     * The colony of the manager.
+     */
+    private final Colony colony;
+
+    /**
+     * Creates the Stat- and AchievementManager for a colony.
+     * @param colony the colony.
+     */
+    public StatisticAchievementManager(final Colony colony)
+    {
+        this.colony = colony;
+    }
+
     /**
      * List of achievements within the colony.
      */
     @NotNull
     private final List<Advancement> colonyAchievements = new ArrayList<>();
 
     @Override
-    public void readFromNBT(@NotNull final NBTTagCompound compound, @NotNull final Colony colony)
+    public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
         // Restore colony achievements
         /*final NBTTagList achievementTagList = compound.getTagList(TAG_ACHIEVEMENT_LIST, Constants.NBT.TAG_COMPOUND);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)